test/governance/TimelockController.test.js

```diff
@@ -1,9 +1,11 @@
-const { BN, constants, expectEvent, expectRevert, time } = require('@openzeppelin/test-helpers');
+const { BN, constants, expectEvent, time, expectRevert } = require('@openzeppelin/test-helpers');
 const { ZERO_ADDRESS, ZERO_BYTES32 } = constants;
 
 const { expect } = require('chai');
 
 const { shouldSupportInterfaces } = require('../utils/introspection/SupportsInterface.behavior');
+const { expectRevertCustomError } = require('../helpers/customError');
+const Enums = require('../helpers/enums');
```

Comment:
We should probably do that, and replace `Enum.OperationState.` with just `OperationState.` for clarity
```suggestion
const { OperationState } = require('../helpers/enums');
```